testng-core/src/main/java/org/testng/internal/FilteredParameters.java

```diff
@@ -54,9 +54,4 @@ public Object[] next() {
     index++;
     return next;
   }
-
-  @Override
-  public void remove() {
-    throw new UnsupportedOperationException("remove");
```

Comment:
Could you confirm that parent behavior is throwing exception?

Reply:
Yes. That's the default implementation in the `Iterator` interface.